### PR TITLE
Feature 76 tabs

### LINKED ISSUE #76: Tabs
as material producer i would like to be able to use tabs

### PR DESCRIPTION
**Pull request by anttileppa**
Resolves #76

### CODE REVIEW

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/pages/components/MaterialTabLayoutComponentFactory.kt [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+                        ),
+                        page = buildContext.page.copy(
+                            resources = tabData.resources
+                        ),
+                        lifecycleListeners = buildContext.lifecycleListeners,
+                        visitorSessionListeners = buildContext.visitorSessionListeners
+                    )
+                )
+
+                tabContentComponent
```
Comment (jarinykanen) [inline app/src/main/java/fi/metatavu/muisti/exhibitionui/pages/components/MaterialTabLayoutComponentFactory.kt:285]: Missing something?

**Review by jarinykanen — CHANGES_REQUESTED** [review]
Check comment

File: app/src/main/java/fi/metatavu/muisti/exhibitionui/pages/components/MaterialTabLayoutComponentFactory.kt (hunk 1 above)
Comment (anttileppa) [reply app/src/main/java/fi/metatavu/muisti/exhibitionui/pages/components/MaterialTabLayoutComponentFactory.kt:285]: Nope. Thats how you return values for Kotlin Lambdas :)

Comment (jarinykanen) [reply app/src/main/java/fi/metatavu/muisti/exhibitionui/pages/components/MaterialTabLayoutComponentFactory.kt:285]: Roger

**Pull request merged**